package.json

```diff
@@ -23,6 +23,7 @@
     "babel-plugin-formatjs": "^10.2.6",
     "blueimp-md5": "^2.10.0",
     "cache-loader": "^4.0.0",
+    "caret-pos": "^2.0.0",
```

Comment:
We already have the similar `jquery-caret-plugin` library.  Neither seems particularly popular; I wonder if we need a library for this or should just be using browser APIs for this (since this seems like a very common thing to want, and thus something there's likely either a much more popular library or just a clean browser API for).  @andersk FYI for the question.

Reply:
I tried out `jquery-caret-plugin`, it doesn't return the `position` of cursor that I need, which is returned by `caret-pos`. Also, `caret-pos` was last updated last year while `jquery-caret-plugin` hasn't been updated in 5years.

Reply:
Yeah, I think removing a library in the other direction might make more sense, especially since we do want to remove jquery dependencies where it's convenient long term.

I'm OK with merging this for now and addressing the problem of too many libraries as a follow-up.